AK/URLParser.cpp

```diff
@@ -177,15 +177,17 @@ URL URLParser::parse(Badge<URL>, StringView const& raw_input, URL const* base_ur
     size_t start_index = 0;
     size_t end_index = raw_input.length();
     for (size_t i = 0; i < raw_input.length(); ++i) {
-        if (0 <= raw_input[i] && raw_input[i] <= 0x20) {
+        signed char ch = raw_input[i];
```

Comment:
Since we now know the exact type, can we just use `i8` and `u8` instead?

Reply:
Sure. :)